doc/installation.rst

```diff
@@ -7,9 +7,27 @@ CadQuery 2 is based on
 `OCP <https://github.com/CadQuery/OCP>`_,
 which is a set of Python bindings for the open-source `OpenCascade <http://www.opencascade.com/>`_ modelling kernel.
 
-Anaconda or Miniconda (Python 3.x editions), Python 3.x
+Conda
 ----------------------------------------------
-CadQuery requires OCP and Python version 3.x
+In principle any conda distrubtion will work, but it is probably best to install `Miniforge <https://github.com/conda-forge/miniforge>`_ to a local directory and to avoid running `conda init`. After performing a local directory installation, Miniforge can be activated via the [scripts,bin]/activate scripts. This will help avoid polluting and breaking the local Python installation. In Linux/MacOS, the local directory installation method looks something like this::
+
+        # Install the script to ~/miniforge
+        wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O miniforge.sh
+        bash miniforge.sh -b -p $HOME/miniforge
+
+        # To activate and use Miniconda
+        source $HOME/miniconda/bin/activate
```

Comment:
Same comment as above. I think that `miniconda` in this line has to change to `miniforge`.

Reply:
done